### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by kristianmandrup**

### CODE REVIEW

File: tmp/__spec-build.js [hunk 1]
```diff
@@ -36,7 +36,17 @@ var _srcHelpersTranslationSynchronizerJs = require('../../src/helpers/translatio
```
Comment (arturmalecki) [inline tmp/__spec-build.js]: Do we need this file in repo?

Comment (kristianmandrup) [inline tmp/__spec-build.js]: Don't think so :P

Comment (kristianmandrup) [inline tmp/__spec-build.js]: also restructure files. Makes no sense to put them all in `/helpers` folder!!

**Comment by arturmalecki** [comment]
@ravirupapara we can't merge it, branch is outdated.

**Pull request closed without merging**